Content.Server/LandMines/LandMineComponent.cs

```diff
@@ -10,4 +10,11 @@ public sealed partial class LandMineComponent : Component
     /// </summary>
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public SoundSpecifier? Sound;
+
+    /// <summary>
+    /// Is the land mine armed and dangerous?
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public bool Armed;
```

Comment:
Probably best to set this to false explicitly as the default!

Reply:
Well it already defaults to false, so it would just be an explicit thing so other programmers recognize it.

Reply:
Yeah exactly! When I first saw it I was a little confused

Reply:
Fixed! Added a false.